courses/views.py

```diff
@@ -78,3 +79,26 @@ def get_queryset(self):
 class RequirementList(generics.ListAPIView, BaseCourseMixin):
     serializer_class = RequirementListSerializer
     queryset = Requirement.objects.all()
+
+
+class UserDetailView(generics.RetrieveAPIView, generics.UpdateAPIView):
```

Comment:
There should be some more restrictions, so that a user can only retrieve / update their own data, I think as this stands, any user can update the info of any other user

Reply:
Would be good to add tests about this to make sure that the API returns `403 Forbidden` in the proper contexts

Reply:
Can users access other users' data?  I overrode the get_object method in the UserView and it is a RetrieveAPIView/UpdateApiView so I don't think you can get other users' data...

Reply:
Does test_multiple_users_independent cover what you're getting at?